standalone/src/models/custom-fields/custom-fields/custom-fields-draft/fields-config.ts

```diff
@@ -0,0 +1,32 @@
+import { fake, type TModelFieldsConfig } from '@/core';
+import { ReferenceDraftRest, ReferenceDraftGraphql } from '@/models/commons';
+import { TCustomFieldsDraftRest, TCustomFieldsDraftGraphql } from '../types';
+
+export const restFieldsConfig: TModelFieldsConfig<TCustomFieldsDraftRest> = {
+  fields: {
+    type: fake(() => ReferenceDraftRest.presets.typeReference()),
+    fields: {},
+  },
+};
+
+export const graphqlFieldsConfig: TModelFieldsConfig<TCustomFieldsDraftGraphql> =
+  {
+    fields: {
+      fields: null,
+      type: fake(() => ReferenceDraftGraphql.presets.typeReference()),
+      typeId: null,
+      typeKey: null,
+    },
+    postBuild: (model) => {
+      if (model.fields) {
+        return {
+          ...model,
+          fields: model.fields.map((field) => ({
+            ...field,
+            value: JSON.stringify(field.value),
+          })),
+        };
+      }
```

Comment:
Could we please leave a comment here about the purpose of this code block for our future selves? 🙏

Reply:
[Too long](https://github.com/commercetools/test-data/pull/863/files#diff-2e1fb93be5eab7508a198d77639adccd8ccdd4c85c13b57620de9711af1d0b5dR22-R35) ? 🤔